clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -42,6 +43,7 @@ object ClioClient extends LazyLogging {
   sealed trait EarlyReturn
   final case class ParsingError(error: String) extends EarlyReturn
   final case class UsageOrHelpAsked(message: String) extends EarlyReturn
+  final case class AuthorizationError(error: String) extends EarlyReturn
```

Comment:
Instead of adding a new error type that doesn't really give us more info, could we just make the existing `ParsingError` more generic? All we need to know is if we should exit with 0 or 1, so distinguishing the types of errors isn't that valuable.